client/my-sites/site-settings/delete-site/index.jsx

```diff
@@ -195,7 +198,12 @@ export const DeleteSite = React.createClass( {
 								}
 							} )
 						}</p>
-					<input className="delete-site__confirm-input" type="text" valueLink={ this.linkState( 'confirmDomain' ) }/>
+					<input
```

Comment:
I'd personally indent the `input` and its attributes one more tab (although the misformatting was not introduced in this PR).